public/script.js

```diff
@@ -19,77 +19,74 @@ const GROUPS = ["<=39", "40-44", "45-49", "50-54", "55-59", "60-64", "65-69", "7
 const GOVERNMENT_TARGET_NUM = 44_000_000;
 const GOVERNMENT_TARGET_PERCENT = GOVERNMENT_TARGET_NUM / 56_286_961;
 
-async function start() {
-    let latestDataDate = await getLatestDataDate();
-    await initializeBarCharts();
-    await initializeLineCharts(latestDataDate);
+function start() {
+    getLatestDataDate().then(initializeLineCharts);
+    initializeBarCharts();
 }
 
-async function initializeBarCharts() {
-    let csv = await d3.csv("latest.csv");
-    makeBarChart(
-        "bar-all",
-        "Percent of England vaccinated",
-        csv.filter(function(row) {
-            return row.group == "all";
-        }),
-        [
-            herdImmunityAnnotation("vertical", "x", true),
-            governmentTargetAnnotation("vertical", "x", true),
-        ],
-        false /* showGroups */);
-    makeBarChart(
-        "bar-over-80",
-        "Percent of >80s" + SUPERSCRIPT_4 + " vaccinated",
-        csv,
-        [],
-        true /* showGroups */);
+function initializeBarCharts() {
```

Comment:
Wouldn't it be cleaner to leave this as `async` (so there's no nested `.then`), and then just don't `await` on the function call? This would allow parallel calls (AFAIU) and not have too much nested code.

Reply:
Yeah you're not wrong, I did consider that but didn't make the change after originally trying to just load the csv once in `start` and then noticing issues due to the mutable nature of JS functions.
Will make that change now.